taskt/Core/Automation/Commands/DataTable Commands/MergeDataTableCommand.cs

```diff
@@ -1,94 +1,91 @@
 ﻿using System;
-using System.Data;
 using System.Collections.Generic;
+using System.Data;
 using System.Linq;
-using System.Xml.Serialization;
 using System.Windows.Forms;
-using taskt.UI.Forms;
-using taskt.UI.CustomControls;
+using System.Xml.Serialization;
+using taskt.Core.Automation.Attributes.ClassAttributes;
+using taskt.Core.Automation.Attributes.PropertyAttributes;
 using taskt.Core.Automation.Engine;
+using taskt.Core.Script;
+using taskt.UI.CustomControls;
+using taskt.UI.Forms;
 
 namespace taskt.Core.Automation.Commands
 {
     [Serializable]
-    [Attributes.ClassAttributes.Group("DataTable Commands")]
-    [Attributes.ClassAttributes.Description("This command allows you to merge Source DataTable into Destination DataTable")]
-    [Attributes.ClassAttributes.UsesDescription("Use this command when you want to merge a DataTable into another DataTable.")]
-    [Attributes.ClassAttributes.ImplementationDescription("")]
+    [Group("DataTable Commands")]
+    [Description("This command merges a source DataTable into a destination DataTable.")]
+
     public class MergeDataTableCommand : ScriptCommand
     {
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please indicate the Destination DataTable Name")]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
-        [Attributes.PropertyAttributes.InputSpecification("Enter an existing DataTable to apply merging into.")]
-        [Attributes.PropertyAttributes.SampleUsage("**myDestDataTable**")]
-        [Attributes.PropertyAttributes.Remarks("")]
-        public string v_Destination_DataTableName { get; set; }
+        [PropertyDescription("Source DataTable")]
+        [InputSpecification("Enter an existing DataTable to merge into another one.")]
+        [SampleUsage("{vSrcDataTable}")]
+        [Remarks("")]
+        [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
+        public string v_SourceDataTable { get; set; }
 
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please indicate the Source DataTable Name")]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
-        [Attributes.PropertyAttributes.InputSpecification("Enter an existing DataTable to merge into another one.")]
-        [Attributes.PropertyAttributes.SampleUsage("**mySrcDataTable**")]
-        [Attributes.PropertyAttributes.Remarks("")]
-        public string v_Source_DataTableName { get; set; }
+        [PropertyDescription("Destination DataTable")]
+        [InputSpecification("Enter an existing DataTable to apply merging into.")]
```

Comment:
This wording is a bit awkward. I would recommend "apply the merge operation to."

Reply:
done